meander/contour_compute.py

```diff
@@ -170,19 +170,20 @@ def _interpolate_simplex_intersections(sample_points, samples, levels, simplex_i
     contours_by_level = []
     for level_i, level in enumerate(levels):
         connected_line_segments = simplex_intersections_by_level[level_i]
-        lines_in_contours = np.unique([line for index_contour in connected_line_segments for line in index_contour], axis=0)
+        if len(connected_line_segments)!=0:
```

Comment:
Can you change this to an "if / else", and in the else block append an empty list to `contours_by_level`? Without this I think the output will be broken, because `contours_by_level` could end up with too few entries.
This may be a little pedantic, but it would be cleaner to compare against the length of the list comprehension passed to `np.unique` rather than the length of `connected_line_segments`. This cuts more at the heart of the bug, and does not rely on the assumption that at least one element of `connected_line_segments` will have non-zero length.